### PR TITLE
Replaces US Indigenous People's Day with International Indigenous People's Day

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Title. This modifies a unit test that tested the holiday I'm removing to test mother's day instead, as International Indigenous People's Day isn't an nth week holiday.

## Why It's Good For The Game
https://i.imgur.com/SkmmNaC.png

## Changelog
:cl:
fix: Replaced US Indigenous People's Day with it's international equivalent
/:cl:

### CODE REVIEW

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -586,6 +586,14 @@
 /datum/holiday/hotdogday/greet()
 	return "Happy National Hot Dog Day!"
 
+/datum/holiday/indigenousUN //Indigenous Peoples' Day from Earth!
```
Comment (Fikou) [inline code/modules/holiday/holidays.dm:589]: dont use uppercase, and it can stay indigenous anyway

Comment (interestingusernam3) [reply code/modules/holiday/holidays.dm]: done

**Review by Mothblocks — DISMISSED** [review]
Test is correct.

**Comment by interestingusernam3** [comment]
where is my stale label

**Comment by interestingusernam3** [comment]
Yeeeeeeeeaaaaaaaaaah

**Pull request merged**